bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/i18n/unifi_nl.properties

```diff
@@ -0,0 +1,143 @@
+# binding
```

Comment:
Please use Crowdin for providing translations.

Reply:
There are many new entries not yet available in crowdin so impossible to add. Besides, since when is it only allowed to use crowdin to do translations? @openhab/add-ons-maintainers did I miss that memo?

Reply:
I think we lack some documentation for the Crowdin processes, but from my understanding:
- Default properties file with English texts flows from the repository to Crowdin.
- After translations are provided in Crowdin they flow the other way through PR's.
- There is no two-way synchronization for translations between the repository and Crowdin - Crowdin is upstream.
- So translations born in the repository will not automatically be made available in Crowdin - manual upload is needed. And I guess some risk is involved in case of conflicts when dealing with existing translations.

@lolodomo - maybe you can shed more light into this or correct me if I'm wrong.

Reply:
I'm resolving this comment, I can upload the Dutch translation to Crowdin after merging this PR and cross fingers that it will not overwrite anything.